src/sidebar/test/frame-sync-test.js

```diff
@@ -189,16 +189,39 @@ describe('FrameSync', function () {
   });
 
   context('when anchoring completes', function () {
+    var clock = sinon.stub();
```

Comment:
Due to the introduction of debouncing, we now need to play god with time in the tests.